<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>932</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	background-color: #F5F5F5;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_x">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>110</y>
      <width>78</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 0;
	background-color: #679B9B;
	color: black;
}

QPushButton:pressed {
	background-color: #477E7E;
}</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sqrt">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>78</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 0;
	background-color: #CFCFCF;
	color: black;
}

QPushButton:pressed {
	background-color: #BDBDBD;
}</string>
    </property>
    <property name="text">
     <string>sqrt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_br1">
    <property name="geometry">
     <rect>
      <x>183</x>
      <y>110</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 0;
	background-color: #84C1C1;
	color: black;
}

QPushButton:pressed {
	background-color: #68AAAA;
}</string>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_br2">
    <property name="geometry">
     <rect>
      <x>248</x>
      <y>110</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 0;
	background-color: #84C1C1;
	color: black;
}

QPushButton:pressed {
	background-color: #68AAAA;
}</string>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pow">
    <property name="geometry">
     <rect>
      <x>313</x>
      <y>110</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 0;
	background-color: #84C1C1;
	color: black;
}

QPushButton:pressed {
	background-color: #68AAAA;
}</string>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_del">
    <property name="geometry">
     <rect>
      <x>378</x>
      <y>110</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 0;
	background-color: #679B9B;
	color: black;
}

QPushButton:pressed {
	background-color: #477E7E;
}</string>
    </property>
    <property name="text">
     <string>del</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_AC">
    <property name="geometry">
     <rect>
      <x>443</x>
      <y>110</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 0;
	background-color: #679B9B;
	color: black;
}

QPushButton:pressed {
	background-color: #477E7E;
}</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_log">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>175</y>
      <width>78</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 0;
	background-color: #CFCFCF;
	color: black;
}

QPushButton:pressed {
	background-color: #BDBDBD;
}</string>
    </property>
    <property name="text">
     <string>log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>183</x>
      <y>175</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 0;
	background-color: #D9D9D9;
	color: black;
}

QPushButton:pressed {
	background-color: #C5C5C5;
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>313</x>
      <y>175</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 0;
	background-color: #D9D9D9;
	color: black;
}

QPushButton:pressed {
	background-color: #C5C5C5;
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>248</x>
      <y>175</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 0;
	background-color: #D9D9D9;
	color: black;
}

QPushButton:pressed {
	background-color: #C5C5C5;
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ln">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>175</y>
      <width>78</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 0;
	background-color: #CFCFCF;
	color: black;
}

QPushButton:pressed {
	background-color: #BDBDBD;
}</string>
    </property>
    <property name="text">
     <string>ln</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mod">
    <property name="geometry">
     <rect>
      <x>378</x>
      <y>175</y>
      <width>125</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 0;
	background-color: #84C1C1;
	color: black;
}

QPushButton:pressed {
	background-color: #68AAAA;
}</string>
    </property>
    <property name="text">
     <string>mod</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mul">
    <property name="geometry">
     <rect>
      <x>378</x>
      <y>240</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 0;
	background-color: #84C1C1;
	color: black;
}

QPushButton:pressed {
	background-color: #68AAAA;
}</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cos">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>240</y>
      <width>78</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 0;
	background-color: #CFCFCF;
	color: black;
}

QPushButton:pressed {
	background-color: #BDBDBD;
}</string>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>313</x>
      <y>240</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 0;
	background-color: #D9D9D9;
	color: black;
}

QPushButton:pressed {
	background-color: #C5C5C5;
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>183</x>
      <y>240</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 0;
	background-color: #D9D9D9;
	color: black;
}

QPushButton:pressed {
	background-color: #C5C5C5;
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_acos">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>240</y>
      <width>78</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 0;
	background-color: #CFCFCF;
	color: black;
}

QPushButton:pressed {
	background-color: #BDBDBD;
}</string>
    </property>
    <property name="text">
     <string>acos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>248</x>
      <y>240</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 0;
	background-color: #D9D9D9;
	color: black;
}

QPushButton:pressed {
	background-color: #C5C5C5;
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_div">
    <property name="geometry">
     <rect>
      <x>443</x>
      <y>240</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 0;
	background-color: #84C1C1;
	color: black;
}

QPushButton:pressed {
	background-color: #68AAAA;
}</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_minus">
    <property name="geometry">
     <rect>
      <x>443</x>
      <y>305</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 0;
	background-color: #84C1C1;
	color: black;
}

QPushButton:pressed {
	background-color: #68AAAA;
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>313</x>
      <y>305</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 0;
	background-color: #D9D9D9;
	color: black;
}

QPushButton:pressed {
	background-color: #C5C5C5;
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>248</x>
      <y>305</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 0;
	background-color: #D9D9D9;
	color: black;
}

QPushButton:pressed {
	background-color: #C5C5C5;
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plus">
    <property name="geometry">
     <rect>
      <x>378</x>
      <y>305</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 0;
	background-color: #84C1C1;
	color: black;
}

QPushButton:pressed {
	background-color: #68AAAA;
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sin">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>305</y>
      <width>78</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 0;
	background-color: #CFCFCF;
	color: black;
}

QPushButton:pressed {
	background-color: #BDBDBD;
}</string>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>183</x>
      <y>305</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 0;
	background-color: #D9D9D9;
	color: black;
}

QPushButton:pressed {
	background-color: #C5C5C5;
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_asin">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>305</y>
      <width>78</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 0;
	background-color: #CFCFCF;
	color: black;
}

QPushButton:pressed {
	background-color: #BDBDBD;
}</string>
    </property>
    <property name="text">
     <string>asin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_atan">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>370</y>
      <width>78</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 0;
	background-color: #CFCFCF;
	color: black;
}

QPushButton:pressed {
	background-color: #BDBDBD;
}</string>
    </property>
    <property name="text">
     <string>atan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_eq">
    <property name="geometry">
     <rect>
      <x>378</x>
      <y>370</y>
      <width>125</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 0;
	background-color: #679B9B;
	color: black;
}

QPushButton:pressed {
	background-color: #477E7E;
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>248</x>
      <y>370</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 0;
	background-color: #D9D9D9;
	color: black;
}

QPushButton:pressed {
	background-color: #C5C5C5;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_tan">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>370</y>
      <width>78</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 0;
	background-color: #CFCFCF;
	color: black;
}

QPushButton:pressed {
	background-color: #BDBDBD;
}</string>
    </property>
    <property name="text">
     <string>tan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_point">
    <property name="geometry">
     <rect>
      <x>313</x>
      <y>370</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 0;
	background-color: #D9D9D9;
	color: black;
}

QPushButton:pressed {
	background-color: #C5C5C5;
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_e">
    <property name="geometry">
     <rect>
      <x>183</x>
      <y>370</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 0;
	background-color: #D9D9D9;
	color: black;
}

QPushButton:pressed {
	background-color: #C5C5C5;
}</string>
    </property>
    <property name="text">
     <string>E</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_range">
    <property name="geometry">
     <rect>
      <x>533</x>
      <y>190</y>
      <width>371</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
	border: 0;
	background-color: #D9D9D9;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>331</width>
       <height>141</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="2">
       <widget class="QLabel" name="label_x_max">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
	color: black;
}</string>
        </property>
        <property name="text">
         <string>X max =</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_x_min">
        <property name="styleSheet">
         <string notr="true">QLineEdit {
	qproperty-alignment: AlignRight;
	border: 0;
	background-color: #F5F5F5;
	color: black;
	padding-left: 5px;
	padding-right: 5px;
}</string>
        </property>
        <property name="text">
         <string>-10</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="lineEdit_y_max">
        <property name="styleSheet">
         <string notr="true">QLineEdit {
	qproperty-alignment: AlignRight;
	border: 0;
	background-color: #F5F5F5;
	color: black;
	padding-left: 5px;
	padding-right: 5px;
}</string>
        </property>
        <property name="text">
         <string>10</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_y_min">
        <property name="styleSheet">
         <string notr="true">QLineEdit {
	qproperty-alignment: AlignRight;
	border: 0;
	background-color: #F5F5F5;
	color: black;
	padding-left: 5px;
	padding-right: 5px;
}</string>
        </property>
        <property name="text">
         <string>-10</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="lineEdit_x_max">
        <property name="styleSheet">
         <string notr="true">QLineEdit {
	qproperty-alignment: AlignRight;
	border: 0;
	background-color: #F5F5F5;
	color: black;
	padding-left: 5px;
	padding-right: 5px;
}</string>
        </property>
        <property name="text">
         <string>10</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_x_min">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
	color: black;
}</string>
        </property>
        <property name="text">
         <string>X min =</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_y_min">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
	color: black;
}</string>
        </property>
        <property name="text">
         <string>Y min =</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_y_max">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
	color: black;
}</string>
        </property>
        <property name="text">
         <string>Y max =</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_x_input">
    <property name="geometry">
     <rect>
      <x>533</x>
      <y>110</y>
      <width>371</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
	border: 0;
	background-color: #D9D9D9;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>0</y>
       <width>301</width>
       <height>61</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label_x_9">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
	color: black;
}</string>
        </property>
        <property name="text">
         <string>X =</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_x_input">
        <property name="styleSheet">
         <string notr="true">QLineEdit {
	qproperty-alignment: AlignRight;
	border: 0;
	background-color: #F5F5F5;
	color: black;
	padding-left: 5px;
	padding-right: 5px;
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_build_3">
    <property name="geometry">
     <rect>
      <x>533</x>
      <y>370</y>
      <width>371</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 0;
	background-color: #679B9B;
	color: black;
}

QPushButton:pressed {
	background-color: #477E7E;
}</string>
    </property>
    <property name="text">
     <string>build graph</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_input">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>887</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
	border: 0;
	background-color: #D9D9D9;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="lineEdit_input">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>641</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
	qproperty-alignment: AlignRight;
	border: 0;
	background-color: #F5F5F5;
	color: black;
	padding-left: 5px;
	padding-right: 5px;
}</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_eq">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>20</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: black;
}</string>
     </property>
     <property name="text">
      <string>=</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_result">
     <property name="geometry">
      <rect>
       <x>687</x>
       <y>9</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	qproperty-alignment: AlignRight;
	border: 0;
	background-color: #F5F5F5;
	color: black;
	padding-left: 5px;
	padding-right: 5px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>932</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Режим</string>
    </property>
    <addaction name="from_calc_to_credit"/>
    <addaction name="from_calc_to_deposit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="from_calc_to_credit">
   <property name="text">
    <string>Кредитный калькулятор</string>
   </property>
  </action>
  <action name="from_calc_to_deposit">
   <property name="text">
    <string>Депозитный калькулятор</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
